development_extensions/oni-dev-extension/extension.js

```diff
@@ -156,6 +160,27 @@ function activate(context) {
     };
     let disposable = vscode.workspace.registerTextDocumentContentProvider('foo', textContentProvider)
     disposable.dispose();
+
+    // Configuration
+
+    const rlsLocation = vscode.workspace.getConfiguration().get("reason_language_server.location");
+    console.error("Configured RLS location: ", rlsLocation);
+
+    const editorFontFamily = vscode.workspace.getConfiguration().get("editor.fontFamily");
+    console.error("Editor Font Family: ", editorFontFamily);
+
+    const dontCrash = vscode.workspace.getConfiguration().get("developer.oni.dont-crash");
```

Comment:
This variable name is confusing for me 😆 

It seems like it's just a test boolean? Maybe `developer.oni.config-test-bool`?